dim_checks/moz-fx-data-shared-prod/telemetry/unified_metrics/dim_checks.yaml

```diff
@@ -0,0 +1,95 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
+    - type: not_null
+      params:
+        columns:
+          - submission_date
+          - client_id
+          - sample_id
+          - activity_segment
+          - normalized_app_name
+          - normalized_channel
+          - country
+          - city
+          - days_seen_bits
+          - days_since_first_seen
+          - days_since_seen
+          - device_model
+          - is_new_profile
+          - first_seen_date
+          - locale
+          - normalized_os
+          - durations
+          - normalized_os_version
+          - app_version
+          - os_version_major
+          - os_version_minor
+          - os_version_patch
+    - type: column_length
+      params:
+        columns:
+          - country
+        condition: "= 2"
+    - type: value_in_set
+      params:
+        columns:
+          - normalized_channel
+        expected_values:
+          - nightly
+          - aurora
+          - release
+          - Other
+          - beta
+          - esr
+    - type: value_in_set
+      params:
+        columns:
+          - activity_segment
```

Comment:
Looking again at the code, the activity segment is calculated using a CASE, so there is no chance for other values. I'd remove this validation, is not necessary.

Reply:
For now it might not hurt, quite clearly anything else here would be unexpected and we might want to know about it?

Reply:
Everything not in the expected values is tagged as 'other'. If you remove it the monitoring effort would be less, your choice!

Reply:
I'd just keep it, at this point it doesn't make a difference to the processing.